api/test/Coevent.Test.csproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <Nullable>enable</Nullable>
-
     <IsPackable>false</IsPackable>
+    <RestoreFallbackFolders>clear</RestoreFallbackFolders>
```

Comment:
Think this resolved an error when vscode attempted to `dotnet restore` on startup.